repo-docs/release-process.md

```diff
@@ -0,0 +1,113 @@
+# Release processes
+
+This repository contains two release processes, which are triggered on Git commits:
+
+## 1. The pre-release process
+
+The purpose of the pre-release process is to publish new changes from the main branch as frequently as possible. This has two main benefits:
+
+1. This tests whether we broke some parts of the release process so we don't get surprises when we need a manual release.
+2. This enables implementers to test a new feature or fix before we trigger a manual release.
+
+To achieve its purpose, the pre-release process is executed on every new commit on the main branch. Additionally, pre-releases do not commit anything to the main branch, which allows them to be deployed even if multiple features are merged on the main branch faster than the CI can release them.
+
+## 2. The manual release process
+
+The purpose of the manual release process is to deploy versions of our packages that we feel confident are safe for implementers to use.
+
+To achieve that purpose, the manual release process is triggered manually whenever we feel that the version currently on the main branch is safe. Additionally, every Wednesday, to ensure we don't forget to regularly trigger manual releases, a request for approval to deploy is triggered.
```

Comment:
Can we revert the order with automate releases as the primary mechanism and defaulting to manual releases if there is any issue etc.